engine/sound/src/test/wscript

```diff
@@ -152,7 +152,12 @@ def build(bld):
                     target = 'test_sound_perf',
                     source = 'test_sound_perf.cpp')
 
-    foo = bld.program(features = 'cxx embed test',
+    extra_features = []
+    if waflib.Options.options.with_asan and bld.env.PLATFORM in ['win32', 'x86_64-win32']:
+        extra_features = ['skip_test']
+        print("Skipping sound test since current version throws an error with ASan")
```

Comment:
We're using a very old OpenAL version on Windows.
For now we'll skip this test, but ultimately I'd like to update the OpenAL driver for Win32.